src/System.Net.HttpListener/src/System/Net/Managed/ChunkedInputStream.cs

```diff
@@ -76,7 +76,7 @@ public override int Read(byte[] buffer, int offset, int count)
             return EndRead(ares);
         }
 
-        public override IAsyncResult BeginRead(byte[] buffer, int offset, int count, AsyncCallback cback, object state)
+        public override IAsyncResult BeginRead(byte[] buffer, int offset, int size, AsyncCallback cback, object state)
```

Comment:
Oh, I see, the existing Windows implementation uses "size" and that gets encoded into the argument of the exception that's thrown.  Ugh.  Ok.